src/policy/mallocspace/metadata.rs

```diff
@@ -3,7 +3,6 @@ use crate::util::constants;
 use crate::util::constants::BYTES_IN_WORD;
 use crate::util::conversions;
 use crate::util::heap::layout::vm_layout_constants::BYTES_IN_CHUNK;
-use crate::util::side_metadata::address_to_meta_address;
```

Comment:
I don't have a strong opinion about this, but you could also add `#[cfg(debug_assertions)]` here, instead of copying the `use` statement twice.

Reply:
I was just thinking that if we put the `use` to where it is actually used and guarded by their flag, that makes the code easier to read. I don't mind doing either though.

Reply:
If the `use` statement was only used in one place, I'd prefer the current implementation, but with two or more copies, I'd slightly prefer guarding the `use` statement directly.

Reply:
Sure. I have made the change.